nixos/modules/services/networking/bind.nix

```diff
@@ -33,29 +33,31 @@ let
       ${cfg.extraConfig}
 
       ${ concatMapStrings
-          ({ name, file, master ? true, slaves ? [], masters ? [], extraConfig ? "" }:
+      (
+        { name, file, master ? true, slaves ? [], masters ? [], extraConfig ? "" }:
```

Comment:
This formatting seems odd to me. Can anyone of the `nixpkgs-fmt` maintainers explain to me whythis is better than the original version?

Reply:
might be related to this?
https://github.com/nix-community/nixpkgs-fmt/issues/126

Reply:
This was fixed with a newer version of nixpkgs-fmt